Gemfile

```diff
@@ -0,0 +1,100 @@
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}.git" }
+
+# ruby version
+ruby '2.7.1'
```

Comment:
2.7.2 is latest, no? https://www.ruby-lang.org/en/news/2020/10/02/ruby-2-7-2-released/

Reply:
Updated